modules/upgrade/pages/deprecated/index.adoc

```diff
@@ -30,12 +49,73 @@ This index helps you to identify deprecated features in Redpanda releases and pl
 | 23.2.1 
 | Use xref:reference:rpk/rpk-cluster/rpk-cluster-partitions-move-status.adoc[`rpk cluster partitions move-status`] instead.
 
-| xref:reference:tunable-properties.adoc#partition_autobalancing_movement_batch_size_bytes[`partition_autobalancing_movement_batch_size_bytes`]
-| 23.2.12
-| Use xref:reference:tunable-properties.adoc#partition_autobalancing_concurrent_moves[`partition_autobalancing_concurrent_moves`] instead.
 
-| xref:23.2@reference:redpanda-operator/index.adoc[Cluster and Console custom resources]
-| 23.2.1
-| Use the xref:23.2@upgrade:deprecated/cluster-resource.adoc[Redpanda resource] instead.
+
+|===
+### Flag changes
+
+|===
+| Flags | Deprecated in | Used in | Details
+| `--admin-api-tls-cert`
+
+`--admin-api-tls-enabled`
+
+`--admin-api-tls-key`
+
+`--admin-api-tls-truststore`
+
+`--brokers`
+
+`--config`
+
+`--password`
+
+`--sasl-mechanism`
+
+`--tls-cert`
+
+`--tls-enabled`
+
+`--tls-key`
+
+`--tls-truststore`
+
+`--user`
+
+|23.2.1
+| xref:reference:rpk/rpk-acl/rpk-acl.adoc[`rpk acl`]
+
+xref:reference:rpk/rpk-cluster/rpk-cluster.adoc[`rpk cluster`]
+
+xref:reference:rpk/rpk-topic/rpk-topic.adoc[`rpk topic`]
+
+xref:reference:rpk/rpk-debug/rpk-debug.adoc[`rpk debug`]
+
+xref:reference:rpk/rpk-group/rpk-group.adoc[`rpk group`]
+
+xref:reference:rpk/rpk-redpanda/rpk-redpanda.adoc[`rpk redpanda`]
+
+
+| Security flags got reworked into the global `-X` flag. See xref:reference:rpk/rpk-x-options.adoc[`rpk -X`] for more info.
+
+| `--client-id`
+
+`--client-secret`
+
+|23.2.1
+| xref:reference:rpk/rpk-cloud/rpk-cloud-byoc-install.adoc[`rpk cloud byoc install`]
+
+| Use X instead.
```

Comment:
```suggestion
| Use `-X` instead.
```

Reply:
This is just a placeholder for the actual value.

Reply:
@r-vasquez from what I got, the flags were removed from `rpk cloud byoc install`. They're still present in `rpk cloud byoc`

Reply:
Ok, let's just specify that 

```
-X cloud.client_id=somestring 
-X cloud.client_secret=somelongerstring
```

Can be used and we are good to go :+1: thanks